checks/burrowsc/burrow.go

```diff
@@ -0,0 +1,236 @@
+package burrowsc
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/hootsuite/healthchecks"
+)
+
+// Burrow consumer group and partition statuses (https://github.com/linkedin/Burrow/wiki/http-request-consumer-group-status)
+const (
+	OK       = "OK"
+	WARN     = "WARN"
+	ERR      = "ERR"
+	STOP     = "STOP"
+	STALL    = "STALL"
+	NOTFOUND = "NOTFOUND"
+)
+
+type LagResponse struct {
```

Comment:
this can be un-exported?

Reply:
Was looking to be consistent with how we're defining response structs elsewhere, https://github.com/hootsuite/healthchecks/blob/d87594e518e5824dabdc9448c6323e6974aed448/about.go#L48 but I can update this.